INAppStoreWindow/INAppStoreWindow.m

```diff
@@ -573,6 +574,30 @@ - (void)mouseDragged:(NSEvent *)theEvent
 		}
 	}
 }
+
+- (CGPoint)convertPoint:(CGPoint)point inWindow:(NSWindow *)window {
```

Comment:
Would be better to make this a static C function instead of an instance method in INAppStoreWindow since this doesn't rely on any window state.